--- libsynce-0.10.0/configure.ac~	2007-08-29 02:48:23.945256622 +0300
+++ libsynce-0.10.0/configure.ac	2007-08-29 02:49:43.167064670 +0300
@@ -143,7 +143,8 @@
 		AC_MSG_WARN([iconv.h not found (INCICONV=$INCICONV)])
 		])
 
-AC_CHECK_LIB(socket, socket, AC_CHECK_LIB(nsl, getpwnam), [])
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(getpwnam, nsl)
 AC_CHECK_LIB(m, floor)
 
 AC_CONFIG_FILES( [lib/synce.h] )
