--- libsynce-0.11/configure.ac~	2008-01-27 02:41:43.000000000 +0200
+++ libsynce-0.11/configure.ac	2008-01-27 02:42:48.187511951 +0200
@@ -145,7 +145,8 @@
 
 saved_CFLAGS="$CFLAGS"
 CFLAGS=`echo $CFLAGS | sed 's/-Werror//'`
-AC_CHECK_LIB(socket, socket, AC_CHECK_LIB(nsl, getpwnam), [])
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(getpwnam, nsl)
 AC_CHECK_LIB(m, floor)
 CFLAGS="$saved_CFLAGS"
 
